### PR TITLE
memegen: pass device data + support inapp

### PR DESCRIPTION
**Pull request by AlexLibs**

### CODE REVIEW

**Comment by ghost** [comment]
Build Failed - please check http://jenkins.memeglobal.com:8080/job/prebid_pull_request/7/

File: modules/memegenBidAdapter.js [hunk 1]
```diff
@@ -47,28 +50,61 @@ var MemeGenAdapter = function MemeGenAdapter() {
...
-    var bidRequest = {
-      id: utils.getUniqueIdentifierStr(),
-      imp: [{
-        id: bidReq.bidId,
-        banner: {
-          w: adW,
-          h: adH
+    var bidRequest;
+    if (isInapp) {
+      bidRequest = {
```
Comment (shlomisut) [inline modules/memegenBidAdapter.js]: can you extract a mehtod of `buildInappRequest` and `buildWebRequst`?

Comment (AlexLibs) [reply modules/memegenBidAdapter.js]: I thought of it, but the passing the values will be much more annoing than just an if (which is very simple and can be even replaced by ternary operator)

Comment (shlomisut) [reply modules/memegenBidAdapter.js]: I prefer to extract a method with global params

**Review by shlomisut — APPROVED** [review]

**Comment by shlomisut** [comment]
retest this please

**Comment by ghost** [comment]
Build Passed

**Pull request merged**